primer.md

```diff
@@ -289,6 +289,28 @@ context, for the purposes of this CloudEvent attribute those meanings are not
 relevant and therefore using `id` for some other purpose beyond uniqueness
 checking is out of scope of the specification and not recommended.
 
+### data
+
+Event consumers can use the `datacontenttype` attribute of a CloudEvent to
+determine what the type is, for example `application/json` or `image/png`. There
+are no restrictions on the encoding format of data that may be included with a
+CloudEvent. Event consumers can expect that the event data is transported
+unaltered due to any encoding used by an SDK to facilitate transport over a
+given protocol. When a CloudEvent is sent by an event producer over a supported
```

Comment:
I wonder if after the sentence on lines 297-299 we should avoid being too specific since the base64 processing can end up confusing people and distracting them from the main point. And instead, what if we added something like:
```
In other words, SDKs should avoid modifying the event data and try to keep the original format sent.
This would include actions such as syntax reformatting. Even though it might appear to not influence
the semantic meaning of the information, the SDK can not be sure if the original format would be
meaningful to the consumer's processing.
```
Not sure we would need the rest of the text.